dasharo-compatibility/platform-hibernation-and-resume.robot

```diff
@@ -37,10 +39,36 @@ HIB001.001 Cyclic platform hibernation and resume (Ubuntu 22.04)
     ...    and resume procedure performed cyclically works correctly
     Skip If    not ${HIBERNATION_AND_RESUME_SUPPORT}    HIB001.001 not supported
     Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    HIB001.001 not supported
+    Skip If    ${PLATFORM_SLEEP_TYPE_SELECTABLE}    HIB001.001 not supported
+    Cyclic Platform Hibernation And Resume (Ubuntu 22.04)
+
+HIB001.002 Cyclic platform hibernation and resume (Ubuntu 22.04) (S0ix)
```

Comment:
hibernation is not really related with the suspend mode AFAIK, @mkopec

Reply:
Ok, I deleted it.